pkg/graveler/ref/db_manager.go

```diff
@@ -606,6 +638,10 @@ func bWorker(ctx context.Context, d *pgxpool.Pool, repository *graveler.Reposito
 		if err != nil {
 			return err
 		}
+		err = sWorker(ctx, d, b.StagingToken)
+		if err != nil {
+			return err
+		}
```

Comment:
This is where the load hides, shouldn't we parallelize on it?

Reply:
Done